src/util/TokenStreamBase.js

```diff
@@ -16,7 +16,7 @@ function TokenStreamBase(input, tokenData){
      * @property _reader
      * @private
      */
-    this._reader = input ? new StringReader(input.toString()) : null;
+    this._reader = input !== undefined ? new StringReader(input.toString()) : null;
```

Comment:
@regseb Could you be more specific about where that is happening?  `git grep TokenStreamBase` doesn't turn that up, although it does uncover `TokenStream.prototype = mix(new TokenStreamBase(), ...)` in `src/css/TokenStream.js` which should perhaps be `TokenStream.prototype = mix(Object.create(TokenStreamBase.prototype), ...)` using more-modern JavaScript convention.

(And in fact I *tried* to invoke `TokenStreamBase` with `null` or `undefined` as the first argument from `tests/css/TokenStream.js`, but failed to: because the pattern is read from the field name of the `patterns` object, it is always stringified.)